### PR TITLE
Update public, opt-out and service rating copy

### PR DESCRIPTION
**Pull request by gsvr**
Highlighted cells only:
https://docs.google.com/a/westerncapelabs.com/spreadsheets/d/1zuEg9DYQKtiKYZZc_ECrRDHAOrh2fPO9r43VyfY_6BU/edit#gid=0

https://docs.google.com/a/westerncapelabs.com/spreadsheets/d/1gmkhX5ZTZ-1iE8i9_RSBkf4Zvnx_Qt-w2cNCmm-n3Sw/edit#gid=0

https://docs.google.com/a/westerncapelabs.com/spreadsheets/d/1JEQz24o8QJkSyRCIMOASllLHw7J1IJOCle5QtXfcDEE/edit#gid=0

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.  I got lazy with adding '+' between lines just for the sake of line lengths in editor.  Let me know if I need to fix this.  Also, there are 2 comments in the spreadsheets (link 1 and 3 above) that we should address before landing.

File: go-app-personal.js [hunk 1]
```diff
@@ -987,7 +987,7 @@ go.app = function() {
         };
 
         self.get_finish_reg_sms = function() {
-            return $("Please dial back in to {{ USSD_number }} to complete the pregnancy registration.")
+            return $("Your session timed out. Please dial back in to *134*550# to complete the pregnancy registration so that you can receive messages.")
```
Comment (imsickofmaps) [inline go-app-personal.js]: We should be inserting config channel in here dynamically

File: src/personal.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ go.app = function() {
         };
 
         self.get_finish_reg_sms = function() {
-            return $("Please dial back in to {{ USSD_number }} to complete the pregnancy registration.")
+            return $("Your session timed out. Please dial back in to *134*550# to complete the pregnancy registration so that you can receive messages.")
```
Comment (imsickofmaps) [inline src/personal.js]: Sorry I mean in here, in source js file.

**Comment by imsickofmaps** [comment]
:+1: landing for POT file creation

**Pull request merged**